Documentation/gettingstarted/rancher.rst

```diff
@@ -0,0 +1,95 @@
+****************************
+Installing Cilium on Rancher
+****************************
+
+Create a New Cluster
+=====================
+
+In Rancher UI, navigate to the Clusters page. In the top right, click on the
+`Add Cluster` box to create a new cluster.
```

Comment:
This is RST, not markdown. The simple backquotes in RST are for “[interpreted text](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#interpreted-text)”, text interpreted in a specific context. Instead, you want inline litterals, with double-backquotes.
```suggestion
``Add Cluster`` box to create a new cluster.
```
Here and at other locations in the document. This should fix most of the complaints from the CI action.